templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -14,9 +12,13 @@ contents: |
   [Service]
   # Need oneshot to delay kubelet
   Type=oneshot
+  # Removal of this file signals firstboot completion
+  ExecStartPre=test ! -f /etc/ignition-machine-config-encapsulated.json
   ExecStart=/usr/local/bin/configure-ovs.sh {{.NetworkType}}
   StandardOutput=journal+console
   StandardError=journal+console
+  Restart=on-failure
+  RestartSec=5
```

Comment:
Polling is not great as a general rule, and I think we can avoid it here.

Reply:
OK

Reply:
Note that configure-ovs already has a retry loop: https://github.com/openshift/machine-config-operator/blob/cf8e35e0cac6b3f77c7dc6c8fab8cd395224dbde/templates/common/_base/files/configure-ovs-network.yaml#L964

It's intentionally slow though to allow time to debug in between failed attempts, so it may not be a great fit for this.

Reply:
OK.
I will modify the script.

Reply:
The firstboot unit depends on this unit.  If this unit never ends, then the firstboot never starts.
Only the original implementation solves the problem (retry through systemd), unless we would like to 
change the dependencies a lot more.